### PR TITLE
Find the IP address of the Guest

### PR DESCRIPTION
**Pull request by bexelbie**
Add functionality to determine and report the guest VM external IP

Vagrantfile - updated to use the CentOS7 box
lib/command.rb - modified to use machine.communicate.execute to find
guest IP

### CODE REVIEW

**Comment by navidshaikh** [comment]
@bexelbie : Duplicate of #24 ?

**Comment by bexelbie** [comment]
@navidshaikh It wasn't supposed to be. I was trying to separate the port and IP fixes. I didn't think through the diff against master :)

**Comment by navidshaikh** [comment]
@bexelbie : So should this be closed then?

**Comment by bexelbie** [comment]
I'd prefer to bring hte IP fix and port fix in via separate patches.  Can we get this one merged first?

File: lib/command.rb [hunk 1]
```diff
@@ -37,7 +37,14 @@ def execute
...
+
+          # Find the guest IP
+          command = "ip route get 8.8.8.8 | awk 'NR==1 {print $NF}'"
+          guest_ip = ""
+          machine.communicate.execute(command) do |type, data|
+            guest_ip << data.chomp if type == :stdout
+          end
+	  
           # Finds the host machine port forwarded from guest docker
           port = machine.provider.capability(:forwarded_ports).key(2376)
```
Comment (navidshaikh) [inline lib/command.rb:49]: `port = 2376`

**Comment by bexelbie** [comment]
changed my mind.  Closing as a dup of #24

**Pull request closed without merging**